server/app.js

```diff
@@ -1,21 +1,21 @@
 import express from 'express';
 import bodyParser from 'body-parser';
+// import user from './routes/user';
 import index from './db';
-import user from './routes/user';
+import auth from './routes/auth';
 import request from './routes/request';
 
+require('dotenv').config();
 
 const app = express();
-const port = 9000;
+const port = 3000;
```

Comment:
this will not work when hosting the app on Heroku

Reply:
Ok, I will look into it

Reply:
will it be better if I save it in  .env file? then I can use process.env.PORT where necessary.